app/components/MapSection.jsx

```diff
@@ -1,49 +1,40 @@
 import React from 'react';
 import {addons} from 'react/addons';
 import {GoogleMap, Marker} from 'react-google-maps';
+import {onSitesWithinRadius, addSiteLocation} from '../utils/geo';
 import '../styles/components/MapSection';
-const {update} = addons;
-
-// Placeholder data for markers
-let markerTest = [{
-  position: {
-    lat: 25.0112183,
-    lng: 121.52067570000001,
-    },
-  key: "Taiwan",
-  defaultAnimation: 2
-}, {
-  position: {
-    lat: 34.0112183,
-    lng: 121.52067570000001,
-    },
-  key: "sdfdsf",
-  defaultAnimation: 2
-}];
 
 export default class MapSection extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      origin: new google.maps.LatLng(41.8507300, -87.6512600),
-      destination: new google.maps.LatLng(41.8525800, -87.6514100),
-      directions: null,
-      markers: markerTest,
+      origin: new google.maps.LatLng(34.01940714979137, -118.4945198893547),
+      markers: [],
     };
+
+    // Bind methods in this section
     this._handle_map_click = this._handle_map_click.bind(this);
-  }
+  } // Constructor
 
   componentDidMount() {
+    onSitesWithinRadius(10, (siteId, latLng) => {
+      let newMarkers = this.state.markers;
+      this.setState({
+        markers: newMarkers.concat([{
+          position: {
+            lat: latLng[0],
+            lng: latLng[1]},
+            key: siteId,
+          defaultAnimation: 2}]
+        )
+      });
+    });
   }
 
   _handle_map_click(event) {
```

Comment:
Please, to stick with convention, rename this with camelCase to `_handleMapClick`.